views/blog/post/__form.php

```diff
@@ -41,5 +42,35 @@
                   placeholder="Content" required><?= Html::encode($body['content'] ?? '') ?></textarea>
     </div>
 
+    <div class="form-group">
+        <label for="AddTag">Add tag</label>
+        <input type="text" class="form-control" id="AddTag" placeholder="Add tag" value="">
+        <?= Html::button(
+    'Add',
+    ['class' => 'btn btn-primary mt-2', 'onclick' => 'addTag()']
```

Comment:
It is better not to use `onclick`. Instead, try addEventListener in `app.js` along with adding ID to the button:

```javascript
document.getElementById("#addTag").element.addEventListener('click', addTagFunction);
```